### PR TITLE
Add Narrator Menu hotkey and Narrator Menu number key listening

### PR DESCRIPTION
**Pull request by boholder**
with https://github.com/khanshoaib3/minecraft-access-i18n/pull/7

Most of the files are changed due to `KeyBindingsHandler.isPressed()` is changed to static method.

### CODE REVIEW

File: common/src/main/java/com/github/khanshoaib3/minecraft_access/features/narrator_menu/NarratorMenu.java [hunk 1]
```diff
@@ -18,26 +22,123 @@
...
+    private static final Boolean[] LAST_RUN_HAS_DONE_FLAG = new Boolean[10];
+
+    static {
+        Arrays.fill(LAST_RUN_HAS_DONE_FLAG, Boolean.TRUE);
+    }
+
+    /**
+     * Should be same order as NarratorMenuGUI.init()
+     */
+    private static final MenuFunction[] MENU_FUNCTIONS = new MenuFunction[]{
```
Comment (khanshoaib3) [inline common/src/main/java/com/github/khanshoaib3/minecraft_access/features/narrator_menu/NarratorMenu.java:46]: @boholder We sohuld also add keypad/numpad keys here.

Comment (boholder) [reply common/src/main/java/com/github/khanshoaib3/minecraft_access/features/narrator_menu/NarratorMenu.java:46]: Done

**Comment by boholder** [comment]
Hey... please remember to update the lang version to `1476de38` (comparator change) or newer one after merging this PR. I won't commit `lang` version in the future in case things like this happen again.

**Comment by khanshoaib3** [comment]
This happened because of a merge conflict in `lang`, it's not likely to happen that often I think.

**Review by khanshoaib3 — APPROVED** [review]

**Pull request merged**